torch/nn/parallel/comm.py

```diff
@@ -162,7 +162,7 @@ def scatter(tensor, devices=None, chunk_sizes=None, dim=0, streams=None, *, out=
           into equal chunks.
         dim (int, optional): A dimension along which to chunk :attr:`tensor`.
           Default: ``0``.
-        streams (Iterable[Stream], optional): an iterable of Streams, among
+        streams (Iterable"[Stream]", optional): an iterable of Streams, among
```

Comment:
I'm not super familiar with sphinx, would `cuda.Stream` be specific enough?

cc @kumpera since I believe you added the cpu streams.